style/about.css

```diff
@@ -0,0 +1,147 @@
+body{
+    background-color: black;
+    width: auto; 
+    min-height:100vh;
+    height: 100vh;
+    margin:0;
+    font-family: helvetica;
+    display:grid;
+    flex-direction: column;
+    font-weight: 300;
+    grid-template-columns: .1fr 1fr 1fr 1fr 1fr 1fr .2fr;
+    grid-template-rows: .4 auto .2%;
+    color:#ebecd1; ;
+    justify-content:space-evenly;
+    align-items: center;
+}
+
+h1{
+    font-weight: lighter;
+}
+
+main #main-photo{
+    max-width: 85vw;
+    max-height: 90vh;
+    /* display: flex; */
+        }
+    
+header{
+background-color: rgb(0, 0, 0);
+text-align: center;
+margin-top: 0em;
+grid-column: 3/6;
+grid-row: 1/2;
+justify-content: space-evenly;
+}
+        
+         
+ul{
+display: flex;
+flex-direction: row;
+justify-content:space-evenly;
+align-items: center;
+
+}
+
+header nav li{
+    font-size: 10pt;
+    background-color: #ebecd1; 
+    padding: .3em;
+    border-radius: 15px;
+    text-decoration: none;
+}
+
+
+li{
+    list-style-type: none;
+}
+
+a{
+    text-decoration: none;
+    font-size: 10pt;
+    color: black;
+}
+
+* :visited{
+    color: rgb(10, 3, 3);
+}
+
+
+main{
+    display: flex;
+    grid-column: 2/ 7;
+    grid-row: 2/3;
+    background-color: rgb(0,0,0);
+    /* max-height: 80vh; */
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction:row;
+    justify-content: space-evenly;
+    align-items: flex-end;
+
+}
+
+p{  
+    max-width: 450px;
+    display:inline-flex;
+   
+}
+section{
+    justify-content: space-evenly;
+    text-align: center;
+    padding: 3em;
+    width: fit-content;
+    /* min-width: 400px; */
+    display:block;
+}
+
+.container {
```

Comment:
I like having a class like this on some root element of related tags, and then using that as part of a descendant selector as you did later.